Firestore/Swift/Source/Codable/SelfDocumentId.swift

```diff
@@ -16,32 +16,37 @@
 
 import FirebaseFirestore
 
-/// A special type used to mark a custom object field to be automatically
-/// populated with the document's referecne when the custom object is created
-/// from a Cloud Firestore document (for example, via
-/// `DocumentSnapshot.data()`).
+/// A value that is populated in Codable objects with a `DocumentReference` by
+/// the FirestoreDecoder when a document is read.
+///
+/// Note that limitations in Swift-compiler generated Codable implementations
+/// prevent using this type wrapped in an Optional. Optional SelfDocumentIDs
+/// are possible if you write a custom init(from: Decoder) method.
```

Comment:
`init(from: Decoder)` should be in backticks.

Reply:
Done.